src/components/Header/Header.jsx

```diff
@@ -16,29 +17,18 @@ const Header = () => {
 
   return (
     <header className={styles.header}>
-      <img src={logo} alt="myteam logo" className={styles["header-logo"]} />
-      <nav className={styles.header - nav}>
-        <ul className={mobileMenu ? "" : styles[hide - mobile - menu]}>
-          <img
-            src={icon_close}
-            alt=""
-            className={styles[icon - close]}
-            onClick={toggleMenu}
-          />
-          <li className={styles[nav - list - item]}>
-            <Link to="/" className={styles[nav - link]}>
-              Home
-            </Link>
+      <img src={logo} alt="myteam logo" className={styles.logo} />
+      <nav className={styles.headerNav}>
+        <ul className={listClass}>
+          <img src={icon_close} alt="" className={styles.icon_close} onClick={toggleMenu} />
+          <li className={styles['nav-list-item']}>
```

Comment:
Hyphenated class names like `nav-list-item` need to use this format, which _MUST_ include quotation marks (e.g., `styles["nav-list-item"]`; either single or double will work) for at least two reasons:
1. Without quotation marks, it won't correctly match the thing it's looking for
2. Without quotation marks, automatic formatting tools can break them when trying to make math operations readable, as you can see in the diff: the formatter turned `styles[nav-list-item]` into `styles[nav - list - item]`, which is broken because the key is pulled apart into multiple pieces